client/src/components/Modals/PitchFeedback/index.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { ReactElement, FC, useState, useEffect } from 'react';
+import { Modal, ModalProps, Icon, Menu } from 'semantic-ui-react';
+import { IPitchFeedback } from 'ssw-common';
+
+import { isError, getPitchFeedback } from '../../../api';
+
+import IndividualTab from './individualTab';
+import QuestionsTab from './questionsTab';
+
+import './styles.scss';
+
+interface PitchFeedbackModal extends ModalProps {
+  pitchId: string;
+}
+
+const PitchFeedbackModal: FC<PitchFeedbackModal> = ({
+  pitchId,
+  trigger,
+}): ReactElement => {
+  const TABS = {
+    QUESTIONS: 'Questions',
+    INDIVIDUAL: 'Individual',
+  } as const;
+  type Tab = typeof TABS[keyof typeof TABS];
+  const [isOpen, setIsOpen] = useState(false);
+  const [feedbacks, setFeedbacks] = useState<IPitchFeedback[]>([]);
+  const [currentTab, setCurrentTab] = useState<Tab>(TABS.QUESTIONS);
+
+  useEffect(() => {
+    if (!isOpen) {
+      return;
+    }
+
+    getFeedback();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
why is eslint disabled for the next line?

Reply:
@zorazrr please remove all of these.